docs/src/maintain/overview.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How ESLint is Maintained
+eleventyNavigation:
+    key: how eslint is maintained
+    parent: maintain eslint
+    title: How ESLint is Maintained
+    order: 1
+
+---
+
+This page explains the different roles and structures involved in maintaining ESLint.
+
+## The ESLint Team
+
+The ESLint team work together to develop and maintain ESLint. The team consists of several different roles, including:
+
+* **Technical Steering Committee (TSC)**: Manage releases, review feature requests, and meet regularly to ensure ESLint is properly maintained.
+* **Reviewers**: Review and implement new features.
+* **Committers**: Review and fix bugs and help triage issues.
+* **Website Team**: Focus specifically on [eslint.org](https://eslint.org).
+* **Support Team**: Help in Discord and discussions.
+
+To learn more about the current team members, refer to the [Team](https://eslint.org/team/) page.
+
+## Organization Structure
+
+TODO: provide a little more info about what the OpenJS foundation actually does vis-a-vis ESLint.
+
+ESLint is part of the [OpenJS Foundation](https://openjsf.org/), a nonprofit organization that supports open-source projects and communities in the JavaScript ecosystem. The OpenJS Foundation provides support and resources to help ensure that projects like ESLint can continue to grow and evolve.
+
+## Funding
+
+ESLint is funded through several sources, including:
+
+* [**Open Collective**](https://opencollective.com/eslint): A platform for financing open source projects.
+* [**GitHub Sponsors**](https://github.com/sponsors/eslint): A platform for funding open source projects associated with Github.
+* [**Tidelift**](https://tidelift.com/subscription/pkg/npm-eslint): A subscription service that lets enterprises manage and fund the open source projects that their organization uses.
+* [**Carbon Ads**](https://www.carbonads.net/open-source): Developer-centric advertising provider used on eslint.org.
+* [**Stackaid.us**](https://simulation.stackaid.us/github/eslint/eslint): Tool that developers can use to allocate funding to the open source projects they use.
+
+ESLint uses this funding for the following purposes:
+
+* Pay team members
+* Fund projects
+* Pay for services that keep ESLint running (web hosting, software subscriptions, etc.)
+* Provide financial support to our dependencies and ecosystem
+
+## Joining the Maintainer Team
+
+ESLint is an open-source project, and anyone can contribute to the project. If you're interested in becoming part of the maintainer team, TODO: what to do?
```

Comment:
This info is actually covered in the [governance doc](https://eslint.org/docs/latest/contribute/governance)

Reply:
I think we can probably say "Stop by Discord and introduce yourself" as an easier way to get people started.